book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_reader :rents
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rents = []
+  end
+
+  def add_rents(rent)
+    @rent.push(rent)
+    rent.book = self
+  end
```

Comment:
- [ ] I think this method should actually create a new rental. A person on a particular date should be able to create a rental for a book. Let's kindly try doing it this way.

 ```.ruby
 def add_rents(person, date)
    Rental.new(date, self, person) # book itself should be sent as a parameter('has-many' associations)
 end
```